Backpack-SwiftUI/MapMarker/Classes/BPKPriceMapMarker.swift

```diff
@@ -57,7 +57,7 @@ public struct BPKPriceMapMarker: View {
     }
     
     private var iconForegroundColor: BPKColor {
-        if case .save = icon, state != .selected {
+        if case .save = icon, state == .unselected {
```

Comment:
Before: Use `surfaceHeroColor` for both `unselected + save icon` and `previousSelected + save icon`
After: Use `surfaceHeroColor` for only `unselected + save icon`